gen.py

```diff
@@ -67,10 +73,45 @@ def generate_single_tag(args: Any):
     repo = Repo.default()
     repo.fetch()
     version = args.version
-    _generate_tag(repo, version)
+    _generate_tag(repo, version, force=args.force)
     _generate_list_of_packages()
 
 
+def _nix_prefetch_git(url: str, rev: str) -> str:
+    return _run(f"nix-prefetch-git {url} --rev {rev} --quiet").decode("utf-8")
+
+
+def _prefetch_output_hashes(cargo_lock: str) -> str:
+    output_hashes = set()
+    for package in tomllib.loads(cargo_lock)["package"]:
+        source = package.get("source", "")
+        if not source.startswith("git+"):
+            continue
+
+        pname = f"{package["name"]}-{package["version"]}"
+        hash_ = output_hash_overrides.get(pname)
```

Comment:
Please add a comment explaining why we need the overrides

Reply:
@grihabor Can you tell me where the working hash came from in the first place? 

For now I'm just going to include the same comment I added to the commit body. If you want something else, let me know.